prac_10/wiki.py

```diff
@@ -0,0 +1,50 @@
+import wikipedia
+
+MENU = """---Menu---
+Choose a search option:
+(P)age Title
+(S)earch Phrase"""
+
+
+def main():
+    print(MENU)
+    choice = input(">>> ").upper()
+    while choice != "":
+        if choice == "P":
+            get_page_data()
+        elif choice == "S":
+            get_search_phrase()
+        else:
+            print("Invalid option.")
+        print(MENU)
+        choice = input(">>> ").upper()
+    print("Thank you!")
+
+
+def get_page_data():
+    title = input("Enter page title: ")
+    try:
+        page = wikipedia.page(title, auto_suggest=False)
+        print(f"Title: {page.title}")
+        print(f"URL: {page.url}")
+        print(f"Summary: {page.summary[:500]}...")
```

Comment:
page.summary is a good option, but try to use page.content to match the sample output better.